src/app/(dashboard)/list/students/[id]/page.tsx

```diff
@@ -116,19 +116,19 @@ const SingleStudentPage = () => {
         <div className="p-4 rounded-md bg-white">
           <h1 className="font-semibold text-xl">Shortcuts</h1>
           <div className="mt-4 flex gap-4 flex-wrap text-sm text-gray-500">
-            <Link href="/" className="p-3 rounded-md bg-sky-100">
+            <Link href={`/list/teachers?classId=${3}`} className="p-3 rounded-md bg-sky-100">
```

Comment:
why are you:

- Making this a literal string.
- Just using a single digit?
```suggestion
            <Link href="/list/teachers?classId=3" className="p-3 rounded-md bg-sky-100">
```

But mostly, why hardcode an ID here?

Reply:
features under development, hardcoded ids just for test , the href will be dynamically generated soon.

seems that I didn't precise what to review and what to not